Doctor-Appointment-App/l_static/assets/js/script.js

```diff
@@ -0,0 +1,360 @@
+/*
+Author       : Dreamguys
+Template Name: Doccure - Bootstrap Template
+Version      : 1.0
+*/
+
+(function($) {
+    "use strict";
+	
+	// Stick Sidebar
+	
+	if ($(window).width() > 767) {
+		if($('.theiaStickySidebar').length > 0) {
+			$('.theiaStickySidebar').theiaStickySidebar({
+			  // Settings
+			  additionalMarginTop: 30
```

Comment:
Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/gh/BuildForSDGCohort2/Team-317-Doctor-Appointment/pullRequest?prid=6207082)